README.md

```diff
@@ -22,11 +22,19 @@ pipenv install
 ```
 
 ## Running
+
 [Install Docker](https://docs.docker.com/engine/installation/)
 ```bash
 docker-compose up
 ```
 
+or run locally (this requires postgres running locally on port 5432):
```

Comment:
From some testing locally you also need to change the `ras-party-db.uri` to be `"postgres://postgres:postgres@localhost:5432/postgres"`

Reply:
Or that should just be the new default maybe?

Reply:
I changed it in the config file to localhost locally. I discussed this with @davidmort and he wanted to leave it with him for now.